packages/apps/hr/src/i18n/en.json

```diff
@@ -45,11 +45,8 @@
   "Hr_TimeUnit_Days": "d",
   "Hr_TimeUnit_Hours": "h",
   "Hr_TimeUnit_Minutes": "min",
-  "Hr_SliceAction_FetchExpenseLines": "fetch expense lines",
-  "Hr_SliceAction_SearchDraftExpense": "fetch draft expenses",
-  "Hr_SliceAction_FetchExpense": "fetch expenses",
-  "Hr_SliceAction_FetchMyExpense": "fetch my expense",
-  "Hr_SliceAction_FetchExpenseToValidate": "fetch expense to validate",
```

Comment:
why ?????????????

Reply:
This was not intended, this must have happened when I added the new translations